File: app/src/main/java/com/example/foot_panel/MainActivity.java
package com.example.foot_panel;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

import com.example.foot_panel.databinding.ActivityMainBinding;
import com.example.foot_panel.dialog.TestDialog;

public class MainActivity extends AppCompatActivity implements View.OnClickListener
{
    private ActivityMainBinding mBinding;

    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        mBinding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(mBinding.getRoot());

        mBinding.btnKeyboard.setOnClickListener(this);
        mBinding.btnFootPanel.setOnClickListener(this);
    }

    @Override
    public void onClick(View v)
    {
        if (v == mBinding.btnKeyboard)
        {
            startActivity(new Intent(this, KeyboardActivity.class));
        } else if (v == mBinding.btnFootPanel)
        {
            startActivity(new Intent(this, FootPanelActivity.class));
        }
    }
}

File: app/src/main/java/com/sd/demo/foot_panel/MainActivity.kt
package com.sd.demo.foot_panel

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.sd.demo.foot_panel.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity(), View.OnClickListener {
    private lateinit var _binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        _binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(_binding.root)
    }

    override fun onClick(v: View) {
        when (v) {
            _binding.btnWindow -> startActivity(Intent(this, WindowActivity::class.java))
            _binding.btnKeyboard -> startActivity(Intent(this, KeyboardActivity::class.java))
            _binding.btnFootPanel -> startActivity(Intent(this, FootPanelActivity::class.java))
            _binding.btnScroll -> startActivity(Intent(this, ScrollActivity::class.java))
        }
    }
}